pyproject.toml

```diff
@@ -105,6 +101,94 @@ test = [
     "coverage",
 ]
 
+[tool.ruff]
+target-version = "py38"  # minimum supported version
+line-length = 88  # same as Black.
+select = [
+    "A",  # flake8-builtins
+    "ANN",  # flake8-annotations
+    "ARG",  # flake8-unused-arguments
+    "B",  # flake8-bugbear
+    "BLE",  # flake8-blind-except
+    "C4",  # flake8-comprehensions
+    "D",  # pydocstyle
+    "E",  # pycodestyle error
+    "EM",  # flake8-errmsg
+    "ERA",  # eradicate
+    "F",  # Pyflakes
+    "FA",  # flake8-future-annotations
+    "FBT",  # flake8-boolean-trap
+    "FIX",  # flake8-fixme
+    "FLY",  # flynt
+    "FURB",  # refurb
+    "I",  # isort
+    "ICN",  # flake8-import-conventions
+    "ISC",  # flake8-implicit-str-concat
+    "PERF",  # perflint
+    "PGH",  # pygrep-hooks
+    "PIE",  # flake8-pie
+    "PL",  # pylint
+    "PT",  # flake8-pytest-style
+    "Q",  # flake8-quotes
+    "RET",  # flake8-return
+    "RSE",  # flake8-raise
+    "RUF",  # ruff specific
+    "SIM",  # flake8-simplify
+    "T10",  # flake8-debugger
+    "T20",  # flake8-print
+    "TCH",  # flake8-type-checking
+    "TRY",  # tryceratops
+    "UP",  # python upgrade
+    "W",  # pycodestyle warning
+    "YTT",  # flake8-2020
+]
+ignore = [
+    "D107",  # missing __init__ docstring, we do that in the class docstring.
+    "D203",  # one blank line before class; we want 2!
```

Comment:
We want no blank lines before the class, don't we?
```python
class Something:
"""blah blah"""
```

What you're describing would look like this:
```python
class Something:


"""blah blah"""
```

Reply:
Yeah, uh, i misunderstood the rule. Ruff said that it was conflicting with another rule that suggested 2 blank lines. 

I'll update the comment to reflect the real world.